Assets/Plugins/Editor/Uplift/UpliftManager.cs

```diff
@@ -689,10 +693,10 @@ private void UpdatePackage(Upset package, TemporaryDirectory td)
 				definition = new DependencyDefinition() { Name = package.PackageName, Version = package.PackageVersion };
 			}
 
-			InstallPackage(package, td, definition, true);
+			InstallPackage(package, td, definition, updateLockfile);
 		}
 
-		public void UpdatePackage(PackageRepo newer, bool updateDependencies = true)
+		public void UpdatePackage(PackageRepo newer, bool updateLockfile = true, bool updateDependencies = true)
```

Comment:
Modifying methods like this is dangerous. Adding a new parameter with a default value before the previous parameter is very risky. I would have added it in third position to make sure no caller accidentally mixes the 2

Reply:
Agreed!

Reply:
Changed in last commits :)